hack/make-rules/run-e2e-tests.sh

```diff
@@ -25,20 +25,97 @@ source "${YURT_ROOT}/hack/lib/build.sh"
 readonly LOCAL_ARCH=$(go env GOHOSTARCH)
 readonly LOCAL_OS=$(go env GOHOSTOS)
 readonly YURT_E2E_TARGETS="test/e2e/yurt-e2e-test"
-KUBECONFIG=${KUBECONFIG:-${HOME}/.kube/config}
+edgeNodeContainerName="openyurt-e2e-test-worker"
+edgeNodeContainer2Name="openyurt-e2e-test-worker2"
+export KUBECONFIG=$HOME/.kube/config
```

Comment:
It seems that we do not need to eport KUBECONFIG here?

Reply:
yes, will remove 'export';